api-driver/ApiDriver.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
-const request = require('request');
+const axios = require('axios');
```

Comment:
@harshaio you used the code from `appmixer-core`? That could be fine (the switch from `request` to `axios` has been tested quite well there).